config/v3_0/types/url.go

```diff
@@ -20,13 +20,11 @@ import (
 	"github.com/vincent-petithory/dataurl"
 
 	"github.com/coreos/ignition/v2/config/shared/errors"
+	"github.com/coreos/ignition/v2/config/util"
 )
 
 func validateURL(s string) error {
 	// Empty url is valid, indicates an empty file
```

Comment:
This comment doesn't seem to apply (in context of this function) with the removal of the if statement. While `url.Parse` won't have an error we will always fall into the default case of the switch statement and return `errors.ErrInvalidScheme`.

Reply:
Oh whoops, yeah that's just a stale comment. Removing.